editor/editor_properties_array_dict.cpp

```diff
@@ -323,100 +356,56 @@ void EditorPropertyArray::update_property() {
 			paginator = memnew(EditorPaginator);
 			paginator->connect("page_changed", callable_mp(this, &EditorPropertyArray::_page_changed));
 			vbox->add_child(paginator);
-		} else {
-			// Bye bye children of the box.
-			for (int i = property_vbox->get_child_count() - 1; i >= 0; i--) {
-				Node *child = property_vbox->get_child(i);
-				if (child == reorder_selected_element_hbox) {
-					continue; // Don't remove the property that the user is moving.
-				}
+		}
 
-				child->queue_free(); // Button still needed after pressed is called.
-				property_vbox->remove_child(child);
-			}
+		// Ensure there is the right amount of slots
```

Comment:
```suggestion
		// Ensure there is the right amount of slots.
```
Per the [comment style](https://docs.godotengine.org/en/latest/contributing/development/code_style_guidelines.html#comment-style-guide).